controller/task_util.py

```diff
@@ -0,0 +1,54 @@
+from typing import List
+from datetime import datetime
+from controller.rats_manager import trigger_rats_creation
+from submodules.model import enums
+from submodules.model.business_objects import general
+from submodules.model.business_objects import tokenization
+from misc.util import put_data_in_minio_bucket, send_websocket_update
+from submodules.model.models import RecordTokenizationTask
+
+
+def set_task_to_started(
+    project_id: str, tokenization_task: RecordTokenizationTask, initial_count: int
+) -> None:
+    tokenization_task.workload = initial_count
+    tokenization_task.state = enums.TokenizerTask.STATE_IN_PROGRESS.value
+    send_websocket_update(
+        project_id, False, ["docbin", "state", str(tokenization_task.state)]
+    )
+    general.commit()
+
+
+def update_tokenization_progress(
+    project_id: str,
+    tokenization_task: RecordTokenizationTask,
+    progress_per_chunk: float,
+) -> None:
+    tokenization_task.progress += progress_per_chunk
```

Comment:
is there a reason why the calculation method of the progress was changed? With this approach, we get a diverging result further into the set.

Reason for this is that we now add a rounded value instead of rounding a calc value 
E.g. chunk size 500 &12345 records will result in: 0.04050... rounded to 3 is 0.041
               new  vs  old
chunk 1: 0.041 vs 0.041
chunk 2: 0.082 vs 0.081004
chunk 20: 0.82 vs 0.81

not a lot and usually corrected by the finalization step (setting to 1) but still :)

Reply:
This was a straight forward calculation i introduced since it is used to display the progress bar in my opinion it is okay like it is but i can ofc set it to the calculation from before. Maybe point for short discussion

Reply:
I would leave it now as it is if it is okay